util/version/version.go

```diff
@@ -201,12 +201,21 @@ func WithBuildTags() CompareOption {
 	}
 }
 
+func WithoutPreReleases() CompareOption {
+	return func(c *comparer) {
+		c.buildTags = false
```

Comment:
```suggestion
		c.withoutPreReleases = true
```
Please add a new field for this flag